### PR TITLE
[Feature] Display mysqld version in motd on Ubuntu

### PR DESCRIPTION
**Pull request by Takios**

### CODE REVIEW

File: percona/motd.sls [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% from "percona/map.jinja" import percona_settings with context %}
+{% set osfullname = salt['grains.get']('osfullname') %}
+
+{% if percona_settings.get('install_motd', False) %}
+{%   if osfullname == 'Ubuntu' %}
```
Comment (martin-helmich) [inline percona/motd.sls:5]: Maybe use `salt['grains.get']('os_family') == 'Debian'` instead?

Comment (Takios) [reply percona/motd.sls:5]: Nope, this does not work on Debian. Debian uses plain old static /etc/motd.

Comment (martin-helmich) [reply percona/motd.sls:5]: :+1: then.

**Review by martin-helmich — APPROVED** [review]

**Comment by luitzifa** [comment]
i know sometime you just wanna get things done, but this formula was build to support redhat based distributions too. so i'd wish it keeps supporting redhat with all new features.

**Pull request merged**